addons/delivery/models/stock_picking.py

```diff
@@ -58,29 +58,29 @@ class StockPicking(models.Model):
     _inherit = 'stock.picking'
 
 
-    @api.one
     @api.depends('move_line_ids')
     def _compute_packages(self):
-        self.ensure_one()
-        packs = set()
-        for move_line in self.move_line_ids:
-            if move_line.result_package_id:
-                packs.add(move_line.result_package_id.id)
-        self.package_ids = list(packs)
+        for rec in self:
+            rec.ensure_one()
```

Comment:
Should be probably moved out of the loop or removed depending on the calls and the original intention to add it.